README.md

```diff
@@ -168,31 +168,31 @@ Ghostfolio is available for various home server systems, including [CasaOS](http
 
 #### Debug
 
-Run `yarn watch:server` and click _Debug API_ in [Visual Studio Code](https://code.visualstudio.com)
+Run `npm run watch:server` and click _Debug API_ in [Visual Studio Code](https://code.visualstudio.com)
 
 #### Serve
 
-Run `yarn start:server`
+Run `npm start:server`
 
 ### Start Client
 
-Run `yarn start:client` and open https://localhost:4200/en in your browser
+Run `npm start:client` and open https://localhost:4200/en in your browser
```

Comment:
`npm run start:client`❓

Reply:
Sorry I forgot to add run. That was really dumb from my side.